Source/Core/VideoCommon/PixelShaderGen.cpp

```diff
@@ -1336,15 +1336,27 @@ ShaderCode GeneratePixelShaderCode(APIType api_type, const ShaderHostConfig& hos
       {
         out.Write("\t\tcustom_data.final_color = float4(ocol0.x / 255.0, ocol0.y / 255.0, ocol0.z "
                   "/ 255.0, ocol0.w / 255.0);\n");
```

Comment:
All of this would probably be easier to understand if `rgba` was used instead of `xyzw` ([these are equivalent](https://www.khronos.org/opengl/wiki/Data_Type_(GLSL)#Swizzling)).

```suggestion
        out.Write("\t\tcustom_data.final_color = float4(ocol0.r / 255.0, ocol0.g / 255.0, ocol0.b "
                  "/ 255.0, ocol0.a / 255.0);\n");
```

Reply:
Ah good idea.  Thanks!